amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseRefresher.java

```diff
@@ -787,6 +802,16 @@ protected DependencyException convertAndRethrowExceptions(String operation, Stri
         }
     }
 
+    private CreateTableRequest.Builder createTableRequestBuilder() {
+    final CreateTableRequest.Builder builder = CreateTableRequest.builder().tableName(table).keySchema(serializer.getKeySchema())
```

Comment:
The indentation for code in this method is incorrect. (Looks like most lines should be indented +1 level?)

Would you kindly fix so the indentation matches the convention of other methods?

(FWIW, I normally regard whitespace as a nitpick. However, seeing `}` at position 1 is jarring when it isn't end-of-class.)

Reply:
Fixed!